testing/helpers.go

```diff
@@ -101,15 +103,44 @@ func GetPermissionsRequestsUseCase(t *testing.T) usecases.PermissionsRequests {
 	return usecases.NewPermissionsRequests(GetRepo(t)).WithContext(context.Background())
 }
 
-// CreateRootServiceAccount creates a root service account with root access
-func CreateRootServiceAccount(t *testing.T) *models.ServiceAccount {
-	return CreateServiceAccountWithPermissions(t, "root", "*::RO::*::*")
+// CreateRootServiceAccountWithKeyPair creates a root service account with root access using KeyPair
+func CreateRootServiceAccountWithKeyPair(t *testing.T) *models.ServiceAccount {
+	t.Helper()
+	return CreateServiceAccountWithPermissions(t, "root", "root@test.com", models.AuthenticationTypes.KeyPair, "*::RO::*::*")
+}
+
+// CreateRootServiceAccountWithOAuth creates a root service account with root access using OAuth
+func CreateRootServiceAccountWithOAuth(t *testing.T) *models.ServiceAccount {
+	t.Helper()
+
+	serviceAccount := CreateServiceAccountWithPermissions(t, "root", "root@test.com", models.AuthenticationTypes.OAuth2, "*::RO::*::*")
+	token := &models.Token{
```

Comment:
Looking at this now, this part it seems to be fit to have its own function, like `func CreateOauthToken` 🤔 💭 Thoughts?

Reply:
if it's working right now, I would do that when neeeded.